QATCH/ui/mainWindow.py

```
@@ -5220,20 +5222,25 @@ def update_resources(self, branch):
+                        if os.path.exists(file_local):
+                            Log.d(f"Deleting {file_local}...")
+                            os.remove(file_local)
+                        Log.d(f"Writing {file_local}...")
                         self._dbx_connection.files_download_to_file(
                             file_local, file_remote)
-                    if Constants.UpdateEngine == UpdateEngines.GitHub:
+                    if Constants.UpdateEngine in [UpdateEngines.GitHub, UpdateEngines.Nightly]:
                         git_mapped_url = Constants.UpdateGitRepo + '/raw/' + \
                             Constants.UpdateGitBranch + \
                             file_remote.replace(branch, "QATCH")
                         response = requests.get(git_mapped_url)
```

Ruff (0.13.3)

Probable use of `requests` call without timeout

(S113)